Separate Moments from math and optimize it

xiaomengy opened the pull request:
Summary:
Separate Moments from math and optimize it

i-am-not-moving-c2-to-c10

Differential Revision: D13742472

houseroad reviewed (approved):
Looks good. Really a good move.

xiaomengy committed:
Separate Moments from math and optimize it (#16175)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16175

Separate Moments from math and optimize it

i-am-not-moving-c2-to-c10

Reviewed By: houseroad

Differential Revision: D13742472

fbshipit-source-id: 6e7c6a7c469ab31661ed2ac1cbb96cb3990499f7
